pkg/cmd/secret/delete/delete_test.go

```diff
@@ -199,6 +208,10 @@ func TestNewCmdDeleteBaseRepoFuncs(t *testing.T) {
 				},
 			}
 
+			for k, v := range tt.env {
+				t.Setenv(k, v)
+			}
```

Comment:
Following the above comment:

```suggestion
			t.Setenv("GH_REPO", tt.ghRepoEnvVar)
```